README.md

```diff
@@ -14,16 +14,38 @@ heroku buildpacks:add https://github.com/moneymeets/python-poetry-buildpack.git
 heroku buildpacks:add heroku/python
```

Comment:
Adding some instructions on how to run tests locally would be nice.

Reply:
Done.